### ISSUE TITLE
Transcription conventions

### ISSUE THREAD

**Issue by meehkal** [langdoc/FRechdoc#2]
Write a document on our transcription conventions. Use HIAT as a source of inspiration.
#### Preliminary notes:
- Numbers are written in words
- … (not three consequent periods)
- Sentences start with capital letter
- Uncertain passage
- Incomprehensible passage
- False start
- Hesitation
- Self correction
#### Preliminary questions:
- Dialectal forms in orth: How much normalized?
  - Not very much, the transcriptions are pretty good presentation of the phonology of the dialects in question
- Quotes in orth: Should we use the right quotes for individual languages or should we rather tag quotes (like in LaTeX)?
  - Further comment below

**Comment by nikopartanen** [comment]
One aspect is that hopefully we will have more Tundra Nenets mixed to Komi one day, this brings additional issues with quotation marks, see utterances kpv_izva20150411-1-b-1045 and kpv_izva20150411-1-b-1046:

> Ӈацекэця" хойку' ниня хабэвкоӈэ питалюрӈа".

So Tundra Nenets uses both " and ' to mark glottal stops. This would be one argument not to use these in the function of quotation marks in the corpus elsewhere. But I don't really know, just wanted to mention this. Lotta Jalava commented some time when I asked about this that in Cyrillic text this is not usually a problem as « » are preferred. In principle those also could be one alternative for us?

**Comment by meehkal** [comment]
Please make such notes in the ticket. It helps us keeping track.

I hope that Unicode differentiates between quotes and letters.

The comment by “One Tundra Nenets Researcher” is stupid, unless s/he uses a typewriter. Who said this?

Micha

**Comment by meehkal** [comment]
these two letters are not quotes
ˮ MODIFIER LETTER DOUBLE APOSTROPHE
ʼ MODIFIER LETTER APOSTROPHE

**Comment by nikopartanen** [comment]
Yes, right, I just changed the characters used in Nenets to those in [that ELAN file as well](https://github.com/langdoc/kpv/commit/18375cc77f967aa413f810af1c02b50ec04f62c9).
